libcontainer/specconv/spec_linux.go

```diff
@@ -508,6 +508,15 @@ func stringToDeviceRune(s string) (rune, error) {
 	}
 }
 
+func deviceIsExist(dev []*configs.Device, path string) bool {
+	for _, d := range dev {
+		if d.Path == path {
```

Comment:
We should use  user's additional devices at first , if not duplicate, than add the pre-configured device.
@cyphar WDYT ?  thanks for your suggestion, I have changed the code.